File: Practica5/Marco teorico/1. Referencias/A. Puntero Implícito this/1.1.java
class Clase {
    private int variableEntera;
    private float variableFlotante;

    public Clase() {
        variableEntera = 0;
        variableFlotante = 0;
    }

    public int getVariableEntera() {
        return this.variableEntera;
    }

    public float getVariableFlotante() {
        return this.variableFlotante;
    }
}

public class Main {
    public static void main(String[] args) {
        Clase objeto = new Clase();
        System.out.println(objeto.getVariableEntera());
        System.out.println(objeto.getVariableFlotante());
    }
}

File: Practica5/Marco teorico/1. Referencias/A. Puntero Implícito this/1.1.cpp
#include <iostream>
using namespace std;
class Clase
{
    int variableEntera;
    float variableFlotante;
    public:
        Clase(){
            variableEntera = 0;
            variableFlotante = 0;
        }
    int getVariableEntera(){
        return this -> variableEntera;
    }
    float getVariableFlotante(){
        return this -> variableFlotante;
    }
    };
    
int main(){
    Clase objeto;
        cout << objeto.getVariableEntera() << endl;
        cout << objeto.getVariableFlotante() << endl;
        system("pause");
    return 0;
}
